### PR TITLE
Ssl support

### PR DESCRIPTION
**Pull request by BRONSOLO**
ping @chriddyp @theengineear

### CODE REVIEW

File: chunked_requests/chunked_request.py [hunk 1]
```diff
@@ -254,13 +254,15 @@ def _isconnected(self):
...
-                return True
+            elif isinstance(e, SSLError):
+                if e.errno == 2:
+                    # errno 2 occurs when trying to read or write data, but more
+                    # data needs to be received on the underlying TCP transport
+                    # before the request can be fulfilled.
+                    #
+                    # Python 2.7.9+ and Python 3.3+ give this its own exception,
+                    # SSLWantReadError
+                    return True
```
Comment (theengineear) [inline chunked_requests/chunked_request.py]: I think you need to `raise e` still `if not e.errno == 2`, right?

Comment (BRONSOLO) [inline chunked_requests/chunked_request.py]: oh yep thx.

Comment (theengineear) [inline chunked_requests/chunked_request.py]: you could:

`elif isinstance(e, SSLError) and e.errno == 2` if ya want

Comment (BRONSOLO) [inline chunked_requests/chunked_request.py]: I just raised : )

**Comment by theengineear** [comment]
💃

**Comment by chriddyp** [comment]
💃

**Pull request merged**